src/main/kotlin/tanvd/grazi/ide/GraziInspection.kt

```diff
@@ -18,73 +20,81 @@ class GraziInspection : LocalInspectionTool() {
     companion object : GraziLifecycle {
         private fun getProblemMessage(fix: Typo): String {
             if (ApplicationManager.getApplication().isUnitTestMode) return fix.info.rule.id
+            return html {
+                if (fix.isSpellingTypo) {
+                    if (fix.info.rule.description.length > 50) {
+                        style = "width: 300px;"
```

Comment:
Check that it works for all resolutions

Reply:
Ok on Mac, later will test on Windows and Linux

Reply:
Ok on other platforms